src/app/malaria-101/malaria-101.component.scss

```diff
@@ -0,0 +1,220 @@
+@import "../styles/main";
+
+video {
+    border: 3px solid $dark-grey;
+    background-color: $black !important;
+}
+
+.pieces {
+    height: 125px;
+    width: 115px;
+    border-radius: 25px;
+}
+
+img {
```

Comment:
do not use `img` if you only want the image styles applied to this specific activity, give it some class instead. `img` will apply to ALL image tags